components/brave_wallet_ui/components/extension/select-address/index.tsx

```diff
@@ -25,10 +26,11 @@ export interface Props {
 export default class SelectAddress extends React.PureComponent<Props> {
   render () {
     const { account, isSelected, action } = this.props
+    const orb = create({ seed: account.address, size: 8, scale: 16 }).toDataURL()
```

Comment:
same issue re: performance

Reply:
Updated this to be a functional component and changed to useMemo Hook!